plugins/BassBooster/BassBooster.cpp

```diff
@@ -102,7 +102,7 @@ bool BassBoosterEffect::processAudioBuffer( sampleFrame* buf, const fpp_t frames
 		m_bbFX.leftFX().setGain( gain );
 		m_bbFX.rightFX().setGain( gain);
 
-		sample_t s[2] = { buf[f][0], buf[f][1] };
+		auto s = std::array {buf[f][0], buf[f][1]};
```

Comment:
Inconsistent formatting(not sure it's enforced or not).
```suggestion
		auto s = std::array{buf[f][0], buf[f][1]};
```